src/client/src/ui/spotTile/components/PriceMovement.tsx

```diff
@@ -10,28 +10,27 @@ interface Props {
 
 const MovementIcon = styled('i')<{ show: boolean; color: string }>`
   text-align: center;
-  color: ${({ theme: { palette }, color }) => palette[color].normal}}};
+  color: ${({ theme, color }) => theme[color].base}}};
```

Comment:
Is there a more general concept we extrapolate?

— or, is there a better shape for our `Theme` type?

Reply:
Yes, I think we could shape things better to suit these types of situations.